src/wp-includes/class-wp-block-bindings-registry.php

```diff
@@ -32,6 +32,19 @@ final class WP_Block_Bindings_Registry {
 	 */
 	private static $instance = null;
 
+	/**
+	 * Allowed block that can use the block bindings API.
+	 *
+	 * @since 6.5.0
+	 * @var WP_Block_Bindings_Registry[]
```

Comment:
Nitpick: This should just be `array`.

Reply:
I think you need only the list of blocks here:

```php
const SUPPORTED_BLOCKS = array(
	'core/paragraph',
	'core/heading',
	'core/image',
	'core/button',
);
```

Reply:
This should be fixed now: [link](https://github.com/WordPress/wordpress-develop/pull/6079/files#diff-3fd1b8d9e55d4fe09b5f48002d360d3ba9d338d6f33ef3959828424a590b4947R35-R46).